src/pybind/mgr/dashboard/frontend/src/app/shared/components/error-panel/error-panel.component.less

```diff
@@ -0,0 +1,11 @@
+@import "../../../../../node_modules/bootstrap/less/variables.less";
```

Comment:
Please change this file to `.scss`.

Reply:
That's not possible because Bootstrap only ships .less files.

Reply:
Fixed